tfhe/src/conformance.rs

```diff
@@ -0,0 +1,28 @@
+/// A trait for objects which can be checked to be conformant with a parameter set
+pub trait ParameterSetConformant {
+    type ParameterSet;
+
+    fn is_conformant(&self, parameter_set: &Self::ParameterSet) -> bool;
+}
+
+/// A constraint on a list size
+/// The list must be composed of a number `n` of groups of size `group_size` which means list size
+/// must be a multiple of `group_size`.
+/// Moreover, `n` must be:
+/// - bigger or equal to `min_group_count`
+/// - smaller of equal to `max_group_count`
+#[derive(Copy, Clone)]
+pub struct ListSizeConstraint {
+    pub min_group_count: usize,
+
+    pub max_group_count: usize,
```

Comment:
could we rename those to min_inclusive... and max_inclusive... ? Don't forget the docstring

Reply:
Done